pkg/config/validator.go

```diff
@@ -14,9 +14,10 @@ import (
 )
```

Comment:
I've just noticed a problem - please modify `Teams` in `config.go` and add `required_if=Enabled true` for BotBindings, similarly like with others. Thanks!

```go
// Teams creds for authentication with MS Teams
type Teams struct {
        // (...)
	Bindings     BotBindings  `yaml:"required_if=Enabled true bindings"`

}

```